### ISSUE TITLE
Copyq isnt opening at all ( No GUI )

### ISSUE THREAD

**Issue by xd003** [hluk/CopyQ#1452]
**Describe the bug**
i have installed the arch package of copyq from here -https://www.archlinux.org/packages/community/x86_64/copyq/
it got installed successfully , but the app never opens  . I tried in terminal too , its just stuck at same place without outputting anything after running copyq .

**Version, OS and Environment**

 - Application Version : CopyQ Clipboard Manager v3.12.0
 - OS : EndeavourOS ( An Arch Linux based distro )
 - Desktop environment : Deepin desktop environment
 - Arch: x86_64-little_endian-lp64
 - Qt: 5.15.0
 - Compiler: GCC

**Further Info** 

Running ```copyq info logs``` returns nothing
Running ```copy show``` does nothing , its get stuck at same place 
Running ```env COPYQ_LOG_LEVEL=DEBUG copyq``` returns the following
```
DEBUG: Server "/home/xd003/.config/copyq/.copyq_s" started.
DEBUG: Geometry: Window "MainWindow": New geometry for "Options/MainWindow_geometry_1366x768"
DEBUG: Geometry: Window "MainWindow": Restore geometry "Options/MainWindow_geometry_1366x768": 645x421,360,173 -> 645x421,360,173
DEBUG: Focus window is "Terminal"
DEBUG: Loading configuration
DEBUG: Configuration loaded
DEBUG: Starting monitor
DEBUG: Executing: copyq --clipboard-access monitorClipboard
DEBUG: Starting callback: onStart
DEBUG: Executing: copyq onStart
DEBUG: Updating tray menu
DEBUG: Tab "&clipboard": Loading items
DEBUG: Tab "&clipboard": 9 items loaded
DEBUG: QtWarning: QXcbConnection: XCB error: 5 (BadAtom), sequence: 486, resource id: 0, major code: 20 (GetProperty), minor code: 0 (:0, )
DEBUG: Executing: copyq eval -- runMenuCommandFilters()
DEBUG: Updating tray menu
DEBUG: Executing: copyq onClipboardChanged
DEBUG: Executing: copyq onClipboardChanged
DEBUG: Executing: copyq onClipboardChanged
DEBUG: Executing: copyq onClipboardChanged
DEBUG: Executing: copyq onClipboardChanged
DEBUG: Executing: copyq onClipboardChanged
DEBUG: New item: Adding
DEBUG: Updating tray menu
DEBUG: Executing: copyq onClipboardChanged
```

**Comment by hluk** [comment]
> Running `copyq info logs` returns nothing

If you are trying to get logs, the command is: `copyq logs`.

> Running `copy show` does nothing , its get stuck at same place

What does "its get stuck at same place" mean? Does it print anything?

Does the main window show up if you use `copyq showAt 0 0`?

**Comment by xd003** [comment]
@hluk yes after i did ```copyq showAt 0 0``` the window finally showed up

**Comment by xd003** [comment]
Also i needed help wtih few commands , i looked a the documentation but couldnt find it quickly . i would be glad if you could tell me the commands for these actions through terminal

1) Check the status of copyq server ( to determine if its active or not )
2) Start the copyq server
3) Stop the copyq server 
4) Print all the items stored in clipboard history in reverse order with and without numbering ( each item should be on a new line obviously )
5) Empty the current clipboard history ( ie delete all items at once )

**Comment by looploooop** [comment]
Same issue with flatpak version (v3.12.0) on Fedora 32 Silverblue.
Starting the application does not open the GUI. Output of journalctl is:

```
Aug 23 19:13:55 systemd[7851]: Started Application launched by gnome-shell.
Aug 23 19:13:55 systemd[7851]: Started flatpak-com.github.hluk.copyq-33052.scope.
Aug 23 19:13:55 systemd[7851]: gnome-launched-com.github.hluk.copyq.desktop-33052.scope: Succeeded.
Aug 23 19:13:55 copyq[33091]: Failed to load module "canberra-gtk-module"
Aug 23 19:13:55 copyq[33091]: Failed to load module "canberra-gtk-module"
```

**Comment by Muscogulus** [comment]
Same issue with version 3.10.0-1 under Ubuntu 20.04. Used the GNOME Software GUI. (I was looking for a clipboard manager and opted for convenience.) The GUI installed CopyQ with `apt`, not `snap`. 

'copyq show` launches the GUI until a command is selected (or typed on command line), whereupon
![Screenshot from 2020-09-09 13-20-39](https://user-images.githubusercontent.com/6090276/92638055-50f24d00-f29f-11ea-99f9-38198a5b64dc.png)
 the GUI disappears. 

CopyQ server is already running when I do `copyq show`

I uninstalled CopyQ with `apt-get purge copyq` and this prompted the removal of `libqt5script5`

At this point I began seeing the following notification whenever I select text: 
![Screenshot from 2020-09-09 13-20-39](https://user-images.githubusercontent.com/6090276/92638127-70897580-f29f-11ea-81d5-929a6f99b056.png)

**Comment by hluk** [comment]
> Same issue with version 3.10.0-1 under Ubuntu 20.04. Used the GNOME Software GUI. (I was looking for a clipboard manager and opted for convenience.) The GUI installed CopyQ with `apt`, not `snap`.

Please use the latest version (there were a lot of fixes between 3.10.0 and now). You can use [the PPA](https://launchpad.net/~hluk/+archive/ubuntu/copyq) if the package is not yet available in your distribution (see https://copyq.readthedocs.io/en/latest/installation.html).

> I uninstalled CopyQ with `apt-get purge copyq` and this prompted the removal of `libqt5script5`
> 
> At this point I began seeing the following notification whenever I select text:
> ![Screenshot from 2020-09-09 13-20-39](https://user-images.githubusercontent.com/6090276/92638127-70897580-f29f-11ea-81d5-929a6f99b056.png)

This is expected: If you uninstall CopyQ and it's still running, the app won't be able `copyq` executable to launch script processes.
